package.json

```diff
@@ -1,14 +1,18 @@
 {
   "private": true,
-  "devDependencies": {
-    "gulp": "^3.8.8"
+  "scripts": {
+    "prod": "gulp --production",
+    "dev": "gulp watch"
   },
-  "dependencies": {
-    "bootstrap-sass": "^3.0.0",
-    "laravel-elixir": "^4.0.0",
+  "devDependencies": {
+    "bootstrap-sass": "^3.3.7",
+    "gulp": "^3.9.1",
+    "laravel-elixir": "^6.0.0-14",
     "laravel-elixir-vueify": "^1.0.2",
-    "sweetalert": "^1.1.3",
     "underscore": "^1.8.3",
     "vue-resource": "^0.7.0"
+  },
+  "dependencies": {
```

Comment:
Is sweet alert the only one we use directly and the others are built in? If not, this is set up incorrectly